redirect/routes.php

```diff
@@ -645,6 +661,9 @@ class Route {
             'pat',
             'cvvm',
             'vt',
+            'all.eat-api',
+            'ref.eat-api',
+            'canteens.eat-api'
```

Comment:
Any reason these need to be linked publically on the website? The will work even without adding them to the "Special" section.

Reply:
Hmm...
If I think again about it, just for sake of completeness.
I think I'm going to remove them again.